locations/spiders/urban_outfitters.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class UrbanOutfitters(SitemapSpider, StructuredDataSpider):
+    download_delay = 10
+    name = "urban_outfitters"
+    item_attributes = {"brand": "Urban Outfitters", "brand_wikidata": "Q3552193"}
+    allowed_domains = ["www.urbanoutfitters.com"]
+    sitemap_urls = ["https://www.urbanoutfitters.com/robots.txt"]
```

Comment:
Sorry, I've confused things here. Sometimes we need to scrape multiple sitemaps, usually because there is a sitemap per region. Having a closer look the sitemaps are split by language. We only need the default [English](https://www.urbanoutfitters.com/store_sitemap.xml) one, not the others eg [French](https://www.urbanoutfitters.com/fr-fr/store_sitemap.xml).

```suggestion
    sitemap_urls = ["https://www.urbanoutfitters.com/store_sitemap.xml"]
```